src/test/resources/data/StudentFeedbackSubmissionEditPageActionTest.json

```diff
@@ -1456,4 +1456,4 @@
         }
         
     }
-}
\ No newline at end of file
+}
```

Comment:
Actually I don't know why this is showing up, I checked the files and there are no new empty lines at the end of the files.